bashbrew/travis.sh

```diff
@@ -37,6 +37,16 @@ fi
 
 export BASHBREW_LIBRARY="$(dirname "$PWD")/library"
 
+if badTags="$(bashbrew list "${repos[@]}" | grep -E ':.+latest.*|:.*latest.+')" && [ -n "$badTags" ]; then
+	echo >&2
+	echo >&2 "Incorrectly 'latest' tags detected:"
```

Comment:
`Inncorrectly 'latest'`? Perhaps that should be `Incorrectly formatted 'latest'` or `Incorrect 'latest'`.